src/rqt_launch/node_controller.py

```diff
@@ -95,12 +95,12 @@ def check_process_status(self):
             # Checks if it should be respawned
             if self._gui.respawn_toggle.isChecked():
                 rospy.loginfo("Respawning _process: {}".format(
-                                                    self._proxy._process.name))
+                    self._proxy._process.name))
```

Comment:
This line doesn't need to be broken

Reply:
The break is necessary to keep it under 79 characters as specified in PEP 8.